projects/canopy/src/lib/filter-buttons/filter-group.component.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/mixins';
+
+.lg-filter-group {
+  display: block;
+  margin-bottom: var(--component-margin);
+
+  .lg-label {
```

Comment:
I might sound a bit fussy on this one, but where possible we try and avoid the components being aware of each other on the css. An easy way round it is to add the [LgMarginDirective](https://legal-and-general-canopy.netlify.app/?path=/story/directives--margin)  and then override the default margin that way. This adds a utility class which helps the components to have a slightly looser coupling. 

```html
  <legend lg-label [attr.for]="id" lgMarginBottom="xs">
    <ng-content></ng-content>
  </legend>
```

Reply:
Done. Will use this in the future in those kind of cases.